### DISCUSSION TITLE
How hide my company mail address?

### DISCUSSION THREAD

**Discussion by andr1972** [community/community#22581] [New to GitHub]
<p>I have git config --global user.email “<a href="mailto:my.name@mycompany.io">my.name@mycompany.io</a>”</p>
<p>Is possible have local config for open source projects.</p>
<p>Moreover: I do several commits with company mail and is pushed to github/gitlab. Is possible change its existed mail addresses?</p>

**Comment by ghost** [comment, marked as answer]
<p>Hey there <a class="mention" href="/borneq">@borneq</a>,</p>
<p>Welcome to the GitHub Community Forum! I’d recommend you check out <a href="https://help.github.com/articles/setting-your-commit-email-address-in-git/" rel="nofollow noopener">this GitHub support article</a> for some info on managing the email used for commits. If you’d like to update the email info of previous commits, the wonderful folks over at GitHub Support have <a href="https://help.github.com/articles/changing-author-info/" rel="nofollow noopener">written a script to make it a breeze</a> 🙂</p>
<p>Let me know how you go!</p>

**Comment by andr1972** [comment]
<p>Thanks, it works! My open source project now hides my company address.</p>
